lib/livebook/intellisense/identifier_matcher.ex

```diff
@@ -241,10 +242,14 @@ defmodule Livebook.Intellisense.IdentifierMatcher do
   end
 
   defp match_sigil(hint, ctx) do
-    for {:function, module, name, arity, "sigil_" <> sigil_name, documentation, signatures, specs} <-
+    for {:function, module, name, arity, :macro, "sigil_" <> sigil_name, documentation,
```

Comment:
SIgils can be non-macro too, although often there is little use case for that. :)